transports/transport-rest/transport-rest-client/src/main/java/org/apache/servicecomb/transport/rest/client/RestTransportClient.java

```diff
@@ -87,6 +85,21 @@ private static HttpClientOptions createHttpClientOptions() {
     return httpClientOptions;
   }
 
+  private static HttpClientOptions createHttp2ClientOptions() {
+    HttpClientOptions httpClientOptions = new HttpClientOptions();
+    httpClientOptions.setMaxPoolSize(TransportClientConfig.getConnectionMaxPoolSize())
+        .setUseAlpn(TransportClientConfig.getUseAlpn())
+        .setHttp2ClearTextUpgrade(false)
```

Comment:
If setting Http2ClearTextUpgrade to false on default, is it possible to access the service using http protocol or we must using https?  Maybe we can add a configuration for this.

Reply:
@wujimin  should I add config with it ? I remember that I have asked you about it.

Reply:
last developer said that must set to false
but he did not make http2 production ready
so we better to confirm this again: 
  producer a is h2
  producer b is h2c
  one consumer can invoke both a and b

Reply:
when I use setHttp2ClearTextUpgrade(true), I find that both h2 and h2c can run as except.

Reply:
We can set default to invoke both of h2 and h2c, but make it configurable is necessary, I think.